boards/BMS/Src/Io/Io_CurrentSense.c

```diff
@@ -1,13 +1,30 @@
 #include <stddef.h>
+#include <math.h>
 #include "Io_CurrentSense.h"
 
-ExitCode Io_CurrentSense_ConvertToLowResolutionMainCurrent(float adc_voltage, float *low_res_main_current)
+// Offset voltage of output 1. Found to be 2.45V through testing
+#define OUTPUT_1_OFFSET (2.45f)
+// Sensitivity of output 1: 40mV/A
+#define OUTPUT_1_SENSITIVITY (40e-3f)
+// Voltage divider between sensor output and adc input.
+#define OUTPUT_1_DIV (2.155f / (1.1f + 2.155f))
+
+// Offset voltage of output 2. Found to be 2.45V through testing
+#define OUTPUT_2_OFFSET (2.45f)
+// Sensitivity of output 2: 6.67mV/A
+#define OUTPUT_2_SENSITIVITY (6.67e-3f)
+// Voltage divider between sensor output and adc input.
+#define OUTPUT_2_DIV (2.155f / (1.1f + 2.155f))
```

Comment:
Continuing off yesterdays discussion, its better to invert the values for `SENSITIVITY` and `DIV` to save some CPU cycles by using multiplication instead of division

Assembly for reference:
![image](https://user-images.githubusercontent.com/32230980/168864511-77ffc029-94a0-4648-aaf8-188d2f2d5e62.png)

Look up VMUL and VDIV in the link below:
https://developer.arm.com/documentation/ddi0439/b/BEHJADED

Reply:
I changed the current divider. But with the sensitivity I would need to divide in the #define anyways. Is this quicker tho? I guess it only runs on build and doesn't need to divide each time we call the function? Does that sound right?

Reply:
Should be done in compile time i belive so i think you're good